omaha_server/crash/senders.py

```diff
@@ -44,16 +44,20 @@ def __init__(self):
         host = getattr(settings, 'LOGSTASH_HOST', None)
         port = getattr(settings, 'LOGSTASH_PORT', None)
         if host and port:
-            self.handler = TCPLogstashHandler(host, int(port))
+            self.handler = TCPLogstashHandler(host, int(port), version=1)
```

Comment:
This is the most important change; without this version 0 is used by default, which is bad.